.github/workflows/lint-chart.yaml

```diff
@@ -0,0 +1,59 @@
+# validate any chart changes under charts directory
+name: Chart Lint
+
+env:
+  HELM_VERSION: v3.8.1
+  KIND_VERSION: v0.12.0
+  K8S_VERSION: v1.23.4
+
+on:
+  push:
+  pull_request:
```

Comment:
pull_request:
    paths:
      - "charts/karmada/**"
It means that only chartfile changed will need to run this CI workflow.

Reply:
You’re quite right.